packages/terra-docs/src/Notice.module.scss

```diff
@@ -53,14 +246,43 @@
       color: var(--terra-docs-notice-deprecation-title-color, #3d4243);
       margin-left: var(--terra-docs-notice-deprecation-title-margin-left, 2.42857rem);
 
-      /* stylelint-disable-next-line selector-max-compound-selectors */
+      /* stylelint-disable-next-line selector-max-compound-selectors, max-nesting-depth */
```

Comment:
It's a bit of a code smell to constantly disable this. Are there not less nested selectors you could write rules around?

Reply:
since these are not styles for our own implementation of the elements (as is the typical case for majority of our components), these are style overrides of existing markdown rendering & styling, this was the level of specificity needed without class injection or adjusting the existing parser in order apply the overrides for these special presentation inside of the notices components.